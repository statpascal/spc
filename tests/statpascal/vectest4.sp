program test5;

procedure p;
    var
	a: vector of int64;
	b: vector of int8;
        c: vector of boolean;
        v: int64;
    begin
	a := intvec (1, 20);
	b := intvec (5, 10);
	a [5] := 10;
	writeln (a);
	writeln (a [b]);
	c := a < 10;
        writeln (c);
        writeln (a [c])
    end;

begin
    p
end.

   