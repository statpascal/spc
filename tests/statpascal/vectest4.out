1 2 3 4 10 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 
10 6 7 8 9 10 
TRUE TRUE TRUE TRUE FALSE TRUE TRUE TRUE TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE 
1 2 3 4 6 7 8 9 
